content/en/docs/tasks/administer-cluster/topology-manager.md

```diff
@@ -49,6 +49,7 @@ The Topology Manager currently:
 
  - Works on Nodes with the `static` CPU Manager Policy enabled. See [control CPU Management Policies](https://kubernetes.io/docs/tasks/administer-cluster/cpu-management-policies/)
  - Works on Pods in the `Guaranteed` {{< glossary_tooltip text="QoS class" term_id="qos-class" >}}
+

```

Comment:
hello @markmc . Are you able to fix a spelling typo too: s/resposible/responsible?
Line 20: `_Topology Manager_ is a Kubelet component that aims to co-ordinate the set of components that are resposible for these optimizations.`

_Topology Manager_ is a `kubelet` component that aims to coordinate the set of components that are responsible for these optimizations.

Reply:
Well spotted, done